transcoder.go

```diff
@@ -175,13 +179,26 @@ func (t *Transcoder) registerRules(rules []*annotations.HttpRule) error {
 }
 
 func (t *Transcoder) registerMethod(handler http.Handler, methodDesc protoreflect.MethodDescriptor, opts *serviceOptions) error {
-	methodPath := "/" + string(methodDesc.Parent().FullName()) + "/" + string(methodDesc.Name())
+	methodPath := methodPath(methodDesc)
 	if _, ok := t.methods[methodPath]; ok {
 		return fmt.Errorf("duplicate registration: method %s has already been configured", methodDesc.FullName())
 	}
```

Comment:
Could the dynamic resolver be initialized here?

Reply:
It _will_ be initialized here.